src/AddExerciseForm.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Button, FormControl, FormGroup, TextField,
+} from '@material-ui/core';
+import {
+  ChangeEvent, useState, FC,
+} from 'react';
+import useLocalStorage from './hooks/useLocalStorage';
+
+const AddExerciseForm: FC = () => {
+  const [formValues, setFormValues] = useState({ description: '', sets: '', reps: '' });
+
+  const [localStorageExercise, setLocalStorageExercise] = useLocalStorage('monday', []);
+
+  const { description, sets, reps } = formValues;
```

Comment:
Looks like these variables are only being used in the `handleFormSubmit` function, I think just pull them out into there.